composeml/label_times.py

```diff
@@ -320,16 +355,9 @@ def infer_type(self):
         """Infer label type.
 
         Returns:
-            LabelTimes : Label Times as inferred type.
+            str : Inferred label type. Can be "continuous" or "discrete".
```

Comment:
I think intuitively I'd expect the logic in `is_discrete` to be here in `infer_type` and then I'd expect `is_discrete` to just check if `label_type == "discrete`.

then every we currently check `if self.label_type == 'discrete'` we'd just replace with `is_discrete`

Reply:
updated logic